src/sqlfluff/utils/reflow/elements.py

```diff
@@ -16,7 +16,8 @@
     TemplateSegment,
     WhitespaceSegment,
 )
-from sqlfluff.core.rules.base import LintFix, LintResult
+from sqlfluff.core.rules import LintFix
+from sqlfluff.core.rules.base import LintResult
```

Comment:
```suggestion
from sqlfluff.core.rules import LintFix, LintResult
```

`LintResult` is re-exported explicitly by `rules\__init__.py` so outside of that module I think we can export both of them directly from `.rules`.